KinWallet/View Controllers and Co./More/HelpCenterViewController.swift

```diff
@@ -65,6 +71,45 @@ final class HelpCenterViewController: WebViewController {
     fileprivate func loadMainPage() {
         loadURL(helpCenterURL)
     }
+
+    fileprivate func presentErrorAlert() {
+        var title = L10n.SupportSubmitError.title
+        var message = L10n.SupportSubmitError.message
+        if errorCount > 0 {
+            title = L10n.SupportSubmitError.AfterRetry.title
+            message = L10n.SupportSubmitError.AfterRetry.message
+        }
+
+        let alertController = UIAlertController(title: title,
+                                                message: message,
+                                                preferredStyle: .alert)
+        if errorCount == 0 {
+            alertController.addAction(title: L10n.SupportSubmitError.retry, style: .default) {
+              self.webView.evaluateJavaScript("submitForm();")
+            }
+        }
+        alertController.addOkAction()
+        present(alertController, animated: true)
+        errorCount += 1
+    }
+
+    fileprivate func onFormPageLoaded() {
+        let userId = User.current?.userId ?? "No user ID"
+        let platform = "\(UIDevice.current.model) \(ProcessInfo().operatingSystemVersionString)"
+        var debug = "false"
```

Comment:
Same here - you can declare `let isDebug: Bool` and set them inside the `#if` and `#else`